intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -720,6 +720,10 @@ \section{Defining and submitting your job}
 part is the number (e.g., ``\jobnumber''); this is a unique identifier for the job
 and can be used to monitor and manage your job.
 
+\underbar{Remark}: the modules that were loaded when you submitted the job won't be loaded
+when the job is started. So you should reload them in your jobscript. This can be
+very tedious but you can use the before mentioned module collections.
```

Comment:
`won't be loaded`  -> `will \emph{not} be loaded`

`So you should reload them in your jobscript` -> `You should always specify the \lstinline|module load| statements that are required for your job in the job script itself.`

`This can be very tedious but` ->
```
\ifusinglmod
To faciliate this, you can use a pre-defined module collection which you can restore using \lstinline|module restore|.
\fi
```